src/main/java/com/uid2/admin/audit/AdminAuditModel.java

```diff
@@ -0,0 +1,55 @@
+package com.uid2.admin.audit;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import io.vertx.core.json.JsonObject;
+
+public class AdminAuditModel implements AuditModel{
+
+    public final String tableActioned;
```

Comment:
Does this represent client key, salt, etc? Can you use an enum?

Reply:
Adding Enum in update